opendevin/core/main.py

```diff
@@ -49,12 +54,26 @@ async def run_agent_controller(
     It's only used when you launch opendevin backend directly via cmdline.
 
     Args:
+        config: The app config.
         task_str: The task to run.
+        max_iterations: The maximum number of iterations to run.
+        max_budget_per_task: The maximum budget per task.
         exit_on_message: quit if agent asks for a message from user (optional)
         fake_user_response_fn: An optional function that receives the current state (could be None) and returns a fake user response.
-        sandbox: An optional sandbox to run the agent in.
+        sandbox: (will be deprecated) An optional sandbox to run the agent in.
```

Comment:
This will be deprecated. I plan to add a `runtime_initialize_fn: Callable[[Runtime], Awaitable[None]] | None = None` in the next PR, so different evaluation benchmarks can add custom logic to initialize a runtime.